_posts/2017/2017-09-05-my-journey-to-become-a-developer.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: My journey to become a developer
+date: "2017-09-05 10:00:00"
+image: blog-header/journey-developer.jpg
+image-position: bottom
+categories: working-at-epages
+authors: ["Christoph"]
+---
+My name is Christoph, and since the beginning of the year I am working as a frontend developer at ePages.
+I originally started at the company in August 2015 with an apprenticeship as a media designer.
+During that time I mostly did website maintenance, some designs in sketch app or photoshop, and the one or other bootstrap website.
+
+## What is my background?
+
+My background was a solid but expandable knowledge of HTML, CSS, and the most basic design principles which I gained in a two-year schooling as a screen designer.
+I learned how to write concepts, how to create screen designs, and how to implement them as a static website.
+Fun fact: before starting out there I did not even know what HTML and CSS was.
+
+## The first few months in the team
+
+Since the beginning of 2017, with the graduation of my apprenticeship in sight, I started out in Team 42 to get a glimpse at what the life of a developer looks like.
```

Comment:
Outside of epages nobody will have a clue what Team 42 does. Maybe you can replace the name and find an explanation. Or leave the name and explain a bit what the team does?